Tools/machines/leonardo-cineca/install_gpu_dependencies.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
#
# Copyright 2023 The WarpX Community
#
# This file is part of WarpX.
#
# Author: Axel Huebl, Marta Galbiati
# License: BSD-3-Clause-LBNL

```

Reply:
Hi @lucafedeli88! Thank you for the suggestion!